Assignment3/calculatingWithFunctions.js

```diff
@@ -0,0 +1,113 @@
+function zero() {
+
+  if(arguments.length>0){// if there is argument then we simply pass value to the function which is in argument
+    return arguments[0](0)
+  }else{ // we will return integer
+    return 0
+  }
+
+}
+function one() {
+
+  if(arguments.length>0){
+    return arguments[0](1)

```

Comment:
You can user ternary as well. Use it next time.

return (arguments.length>0 ? arguments[0](0) :  0)

Reply:
Sure